src/client/components/Profile/Profile.styling.css

```diff
@@ -85,18 +95,31 @@
   padding: 10% 20%;
 }
 
-.name-edit-btn, .phone-info, .email-info {
+.name-edit-btn,
+.phone-info,
+.email-info {
   align-items: baseline;
 }
 
-.user-name input, .phone-info input {
+.user-name input,
+.phone-info input {
   padding-right: 2%;
   line-height: 1px;
+  border-color: gray;
+}
+
+.profile-sign-out-button {
+  display: flex;
+  justify-content: flex-end;
+}
+
+button {
```

Comment:
Do not use global style here, you will change all buttons in the entire application.

Reply:
fixed them using class names for sign-out and edit buttons.